xml/System/String.xml

```diff
@@ -10567,7 +10580,9 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
   
  Because this method returns the modified string, you can chain together successive calls to the <xref:System.String.Replace%2A> method to perform multiple replacements on the original string. Method calls are executed from left to right. The following example provides an illustration.  
```

Comment:
this note with the interactive sample is repeated in both overloads. It would make more sense to move it to the remarks of the membergroup element as well and remove from the overloads.

Reply:
Done.